config/config_embed.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	_ "embed"
+)
+
+// MainnetConfig is the Neo N3 mainnet configuration.
+//
+//go:embed protocol.mainnet.yml
+var MainnetConfig []byte
+
+// TestnetConfig is the Neo N3 testnet configuration.
+//
+//go:embed protocol.testnet.yml
+var TestnetConfig []byte
+
+// UnitTestnetConfig is the unit testnet configuration.
+//
+//go:embed protocol.unit_testnet.yml
+var UnitTestnetConfig []byte
```

Comment:
Do we need this one, btw?

Reply:
but we have it as hidden flag in cli.
`	cli.BoolFlag{Name: "unittest", Hidden: true}, `

Reply:
I think it's only developers who use this option, and we always have the real config with us.